docs/source/sycamore/transforms/merge.md

```diff
@@ -0,0 +1,44 @@
+## Merge
+
+The merge transform is responsible for 'merging' elements into larger 'chunks'.
+
+The merge transform takes a single argument -- the `merger`, which contains the logic defining which elements to merge and how to merge them. The available mergers are listed below. More information can be found in the {doc}`API documentation </sycamore/APIs/low_level_transforms/merge_elements>`
+
+### Greedy Text Element Merger
+
+The `GreedyTextElementMerger` takes a tokenizer and a token limit, and merges elements together, greedily, until the combined element will overflow the token limit, at which point the merger starts work on a new merged element. If an element is already too big, the `GreedyTextElementMerger` will leave it alone.
+
+For example, using a `CharacterTokenizer` with `max_tokens=4`, you would have the following behavior:
+
+```
+A BC D EF GHI J KLMNO -> ABCD EF GHIJ KLMNO
+```
+
+To add this to a script:
+
+```python
+merger = GreedyTextElementMerger(tokenizer=HuggingFaceTokenizer("sentence-transformers/all-MiniLM-L6-v2", max_tokens=512))
+merged_docset = docset.merge(merger=merger)
+```
+
+### Greedy Section Merger
+
+The `GreedySectionMerger` behaves similarly to the `GreedyTextElementMerger` in that is takes a tokenizer and token limit, and merges elements greedily, stopping before the token limit. The `GreedySectionMerger` adds to this by also starting new elements at section headers, and handling images and tables more gracefully.
+
```

Comment:
I'm a little confused by this explanation. I might say: Like the `GreedyTextElementMerger`, the `GreedySectionMerger` takes a tokenizer and token limit, and merges elements greedily, stopping before the token limit. However, the `GreedySectionMerger` also has additional heuristic rules.

Dan needs to write what these rules are.